zerver/lib/html_diff.py

```diff
@@ -6,7 +6,7 @@
 
 def highlight_html_differences(s1: str, s2: str, msg_id: Optional[int] = None) -> str:
     retval = htmldiff(s1, s2)
-    fragment = lxml.html.fromstring(retval)
+    fragment = lxml.html.fragment_fromstring(retval, create_parent=True)
```

Comment:
Why is `create_parent=True` correct?

Reply:
Without it, `fragment_fromstring` also throws an exception on empty documents:
```
In [1]: import lxml

In [2]: lxml.html.fragment_fromstring("")
---------------------------------------------------------------------------
ParserError                               Traceback (most recent call last)
<ipython-input-2-4c08477dbd71> in <module>
----> 1 lxml.html.fragment_fromstring("")

/srv/zulip-py3-venv/lib/python3.6/site-packages/lxml/html/__init__.py in fragment_fromstring(html, create_parent, base_url, parser, **kw)
    845
    846     if not elements:
--> 847         raise etree.ParserError('No elements found')
    848     if len(elements) > 1:
    849         raise etree.ParserError(

ParserError: No elements found

In [3]: lxml.html.fragment_fromstring("", create_parent=True)
Out[3]: <Element div at 0x7f48ae355408>
```

Reply:
And it's side effect is mostly adding a wrapping div around our diffs output, which seems harmless.  OK.